monoio-http/src/common/body.rs

```diff
@@ -94,6 +104,81 @@ impl<T: Body<Data = Bytes>> BodyExt for T {
             }
         }
     }
+
+    fn decode_content(self, encoding: String) -> Self::BytesFuture {
+        async move {
+            let buf = self.bytes().await?;
+            match encoding.as_str() {
+                "gzip" => {
+                    let mut decoder = GzDecoder::new(buf.as_ref());
+                    let mut decompressed_data = Vec::new();
+                    decoder.read_to_end(&mut decompressed_data)?;
+                    Ok(bytes::Bytes::from(decompressed_data))
+                }
+                "deflate" => {
+                    let mut decoder = ZlibDecoder::new(buf.as_ref());
+                    let mut decompressed_data = Vec::new();
+                    decoder.read_to_end(&mut decompressed_data)?;
+                    Ok(bytes::Bytes::from(decompressed_data))
+                }
+                "br" => {
+                    let mut decoder = brotli::Decompressor::new(buf.as_ref(), 4096);
+                    let mut decompressed_data = Vec::new();
+                    decoder.read_to_end(&mut decompressed_data)?;
+                    Ok(bytes::Bytes::from(decompressed_data))
+                }
+                _ => {
+                    // Unsupported or no encoding, return original data
+                    Ok(buf)
+                }
+            }
+        }
+    }
+
+    fn encode_content(self, accept_encoding: String) -> Self::BytesFuture {
+        async move {
+            let buf = self.bytes().await?;
+            let accepted_encodings: Vec<String> = accept_encoding
+                .split(',')
+                .map(|s| s.trim().to_string())
+                .collect();
+            let supported_encodings = vec!["gzip", "br", "deflate"];
```

Comment:
Use `["gzip", "br", "deflate"]` instead of vec. Or define it as a const array.

Reply:
Changed to const array